packages/conan/recipes/README.md

```diff
@@ -5,3 +5,9 @@ but with quite a few modifications to allow the VFX versions to build, and with
 ASWF-made packages and avoid ABI mixes with the Conan Center packages.
 
 As the Conan Center Index is MIT licensed the whole subfolder here is also MIT licensed for consistency.
+
+## Adding new Conan packages
+
+Follow the great instructions there: [Conan Center Index - How to add Packages](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md).
+
+Then ensure the ASWF-specific settings are added in the `conanfile.py` such as: `python`, `devtoolset`, `ci_common` and `vfx_platform`.
```

Comment:
The Conan support is very new and needs more documentation in general, but would it be possible to add a sentence or two about how '(latest)' ends up translated to the latest version for the VFX platform year (and thus what was previously pulled from the environment variables)? Environment Variables are often a pretty kludgy / fragile (but difficult to avoid) way to pass configuration information around, so this is definitely better.

Reply:
I promise I'll do a good pass as soon as it is stabilised!